pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/source.toolkit.fluxcd.io/v1/GitRepository/customizations.yaml

```diff
@@ -0,0 +1,100 @@
+                return true
+              end
+            end
+          end
+          return false
+        end
+    statusAggregation:
+      luaScript: >
+        function AggregateStatus(desiredObj, statusItems)
+          if statusItems == nil then
+            return desiredObj
+          end
+          if desiredObj.status == nil then
+            desiredObj.status = {}
+          end
+          if desiredObj.status.conditions == nil then
+            desiredObj.status.conditions = {}
+          end
+          artifact = {}
+          lastHandledReconcileAt = ''
+          conditions = {}          
+          local conditionsIndex = 1
+          for i = 1, #statusItems do
+            if statusItems[i].status ~= nil and statusItems[i].status.artifact ~= nil then
+              artifact = statusItems[i].status.artifact
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.lastHandledReconcileAt ~= nil then
+              lastHandledReconcileAt = statusItems[i].status.lastHandledReconcileAt
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.conditions ~= nil then
+              for conditionIndex = 1, #statusItems[i].status.conditions do
+                statusItems[i].status.conditions[conditionIndex].message = statusItems[i].clusterName..': '..statusItems[i].status.conditions[conditionIndex].message
+                conditions[conditionsIndex] = statusItems[i].status.conditions[conditionIndex]
+                conditionsIndex = conditionsIndex + 1
+              end
+            end
+          end
+          desiredObj.status.observedGeneration = desiredObj.metadata.generation
+          desiredObj.status.artifact = artifact
+          desiredObj.status.lastHandledReconcileAt = lastHandledReconcileAt
+          desiredObj.status.conditions = conditions
+          return desiredObj
+        end
+    retention:
+      luaScript: >
+        function Retain(desiredObj, observedObj)
+          if observedObj.spec.suspend ~= nil then
+            desiredObj.spec.suspend = observedObj.spec.suspend
+          end
+          return desiredObj
+        end
+    statusReflection:
+      luaScript: >
+        function ReflectStatus (observedObj)
+          status = {}
+          if observedObj == nil or observedObj.status == nil then 
+            return status
+          end
+          status.conditions = observedObj.status.conditions
+          status.artifact = observedObj.status.artifact
+          status.observedIgnore = observedObj.status.observedIgnore
+          status.observedRecurseSubmodules = observedObj.status.observedRecurseSubmodules
+          status.lastHandledReconcileAt = observedObj.status.lastHandledReconcileAt
+          return status
+        end
+    dependencyInterpretation:
+      luaScript: >
+        function GetDependencies(desiredObj)
```

Comment:
`.status.verify`  specifies the configuration ( which includes `secretRef` that specifies the secret containing the public keys of trusted Git authors ) to verify the Git commit signature(s), it need to be propagated as dependencies.